listener/src/main.rs

```diff
@@ -210,12 +215,13 @@ fn async_main() -> impl Future<Item = (), Error = ()> + Send + 'static {
         SubgraphVersionSwitchingMode::Instant
     ));
 
+    let subgraph_name = SubgraphName::new("dfusion").unwrap();
```

Comment:
why here the subgraph is hardcoded and some lines above is loaded from a constant?

Reply:
Should have been the constant here. Good catch.